packages/dashboard-frontend/src/services/workspace-client/devworkspace/converters/index.ts

```diff
@@ -16,7 +16,7 @@ export const devworkspaceVersion = 'v1alpha2';
 export const devWorkspaceApiGroup = 'workspace.devfile.io';
 export const devworkspaceSingularSubresource = 'devworkspace';
 
-export function devfileToDevWorkspace(devfile: devfileApi.Devfile, routingClass: string, started: boolean): devfileApi.DevWorkspace {
+export function devfileToDevWorkspace(devfile: devfileApi.Devfile, routingClass: string | undefined, started: boolean): devfileApi.DevWorkspace {
```

Comment:
```suggestion
export function devfileToDevWorkspace(devfile: devfileApi.Devfile, routingClass: string, started: boolean): devfileApi.DevWorkspace {
```
it must be always specified as `che` and never undefined.

Reply:
done